apps/schematic/api/schematic_api/controllers/schema_controller.py

```diff
@@ -5,13 +5,29 @@
 from typing import Union
 
 from schematic_api.models.basic_error import BasicError  # noqa: E501
+from schematic_api.models.connected_nodes_page import ConnectedNodesPage  # noqa: E501
 from schematic_api.models.node_properties_page import NodePropertiesPage  # noqa: E501
 from schematic_api.models.nodes_page import NodesPage  # noqa: E501
 from schematic_api.models.validation_rules_page import ValidationRulesPage  # noqa: E501
 from schematic_api import util
 from schematic_api.controllers import schema_controller_impl
 
 
+def get_connected_nodes(schema_url, relationship_type):  # noqa: E501
+    """Gets a subgraph containing all relationships of a given type
```

Comment:
sorry I didn't catch this earlier, but for this endpoint, it does call function that returns a subgraph, but the endpoint itself it returning a list(?) of node pairs.